extensions/strategies/ichi/strategy.js

```diff
@@ -0,0 +1,143 @@
+var z = require('zero-fill')
+, n = require('numbro')
+
+/**
+ * Calculate if the Kumo can be reversed (sideway market or kumo twist)
+ * */
+function isReversable(s){
+	var kumo_size = s.period.close*0.005;
+	//last_kumo = Math.abs(s.period.ssa-s.period.ssb)<kumo_size
```

Comment:
Please remove code what your not use.

Reply:
this code will/should be use, i'm trying to implement the kumo analysis (a bit harder than what I originally thought).